src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -93,7 +93,7 @@ public function __construct(
         }
 
         if ($options) {
```

Comment:
shoudn't this be `if (\is_array($options)) {` everywhere?

Reply:
I agree, otherwise an empty array will not trigger the deprecation. Should this be taken care in a separate PR or this one's fine?

Reply:
this one is fine